README.md

```diff
@@ -3,11 +3,45 @@
 This project is the source for the new metafacture blog.
 This project was created with the gatsby template [gatsby-starter-blog](https://github.com/gatsbyjs/gatsby-starter-blog).
 
+## Installation
+
+Gatsby is build with Node.js.
+
+### Install Node.js
+
+#### Windows
+
+Download and install the latest Node.js version from [the official Node.js website]( https://nodejs.org/en/).
+
+#### Unix
+
+Download the lastest nvm version.
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash
+```
+
+Set default Node.js version. When nvm is installed, it does not default to a particular node version. You’ll need to install the version you want and give nvm instructions to use it. This example uses the version 10 release, but more recent version numbers can be used instead.
+
+```
+nvm install 10
+nvm use 10
+```
+
+### Install gatsby
+
+```
+npm install -g gatsby-cli
+```
+
+npm install
```

Comment:
```suggestion

```

Isn't this here by accident

Reply:
Yes! I think that was part of my 'notes'.